.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Word Desktop (Edge Chromium)",
+            "type": "edge",
+            "request": "attach",
+            "useWebView": "advanced",
+            "port": 9229,
+            "timeout": 600000,
+            "webRoot": "${workspaceRoot}",
+            "preLaunchTask": "Debug: Word Desktop",
+            "postDebugTask": "Stop Debug"
```

Comment:
```
"preLaunchTask": "Debug: Word Desktop", 
"postDebugTask": "Stop Debug"
```
This is of no use if `tasks.json` is not present, So you may either delete this or add the `taks.json` file

Reply:
@Aniruddha-Shriwant 
I think it would be better to remove them.
Since we do not do much tasks.

Reply:
@K-Kumar-01 you may add the following `tasks.json` file 
```{
    // See https://go.microsoft.com/fwlink/?LinkId=733558
    // for the documentation about the tasks.json format
    "version": "2.0.0",
    "tasks": [
        {
            "label": "Debug: Word Desktop",
            "type": "npm",
            "script": "start:desktop -- --app word",
            "presentation": {
                "clear": true,
                "panel": "dedicated",
            },
            "problemMatcher": []
        },
        {
            "label": "Stop Debug",
            "type": "npm",
            "script": "stop",
            "presentation": {
                "clear": true,
                "panel": "shared",
                "showReuseMessage": false
            },
            "problemMatcher": []
        }
    ]
}
```

Reply:
@Aniruddha-Shriwant 
I think it would be better to remove the tasks.json as they are also running up the development server as from what I understand. What do you think?

Reply:
Only removing the scripts also doesn't give the error.

Reply:
> Only removing the scripts also doesn't give the error.

Yes this too works, I just suggested the `tasks.json` because `preLaunchTask and postLaunchTask` were present in `launch.json`'s config.

Reply:
> @Aniruddha-Shriwant
> I think it would be better to remove the tasks.json as they are also running up the development server as from what I understand. What do you think?

Yes if tasks.json are included they will open the new terminal every time they run 
![image](https://user-images.githubusercontent.com/58624108/118369410-d95c8680-b5c0-11eb-8b68-766b9dbbcab2.png)